Sources/Extensions/UITableView/UITableView+Extensions.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Copyright (c) 2018 Touch Instinct
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import UIKit
+
+public extension UITableView {
+
+    /// Adds colored background to the top of the tableview.
+    ///
+    /// - Parameters:
+    ///   - color: header background color
+    func addHeaderBackground(with color: UIColor) {
+        let backgroundView = UIView()
+        backgroundView.backgroundColor = color
+        addSubview(backgroundView)
+        backgroundView.translatesAutoresizingMaskIntoConstraints = false
+        backgroundView.leftAnchor.constraint(equalTo: leftAnchor).isActive = true
```

Comment:
https://github.com/TouchInstinct/LeadKit/blob/master/Sources/Extensions/UIView/UIView%2BLayout.swift#L50
you can change modifier to `internal` (just remove `private`) and use it here)

Reply:
or use https://github.com/TouchInstinct/LeadKit/blob/master/Sources/Extensions/UIView/UIView%2BLayout.swift#L32 with nil size argument

Reply:
Not the same. This view placed above the view.

Reply:
addSubview(backgroundView)
backgroundView.superview == self
right?

Reply:
This view *placed above*, origin == (0, -tableView.height)

Reply:
ok. got it